c2corg_api/views/document_changes.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+from operator import or_, and_
```

Comment:
Shouldn't it rather be
``from sqlalchemy.sql.expression import or_, and_``
?
As in https://github.com/c2corg/v6_api/blob/master/c2corg_api/views/feed.py#L16